test/integration/features/profile.feature

```diff
@@ -2,104 +2,161 @@
 Feature: Profile
   As a user I can perform basic operations of Minishift with profile feature
 
-  Scenario: Starting Minishift with default profile
-     Given Minishift has state "Does Not Exist"
-      When executing "minishift start" succeeds
-      Then Minishift should have state "Running"
-
-Scenario: User should be able to list default profile 'minishift'
-   Given Minishift has state "Running"
-    When executing "minishift profile list" succeeds
-    Then stdout should contain
-    """
-    - minishift	Running		(Active)
-    """
-  Scenario: Getting default profile internal docker registry address
-     Given Minishift has state "Running"
-      When executing "minishift openshift registry" succeeds
-      Then stdout should be valid IP with port number
+  Scenario Outline: As user, I cannot create profile with blank profile name
```

Comment:
nitpick - "As user, I cannot create a profile..."

Same applies to the following couple of scenarios.

Reply:
Hmm :) I took sometime to get a better scenario description infact I got of some them however non of them were convincing

Reply:
The descriptions are good now. Reads quite well now imo. Well done. This really just a typo.